src/components/player/DisplayQuestion.tsx

```diff
@@ -0,0 +1,34 @@
+import { state$ } from "../../state";
+
+function DisplayQuestion() {
+    const question = state$.quiz.question.get();
+    const answers = state$.quiz.answers.get();
+
+    const handleSubmitAnswer = (answer: string) => {
+        console.log(`selected answer: ${answer}`);
+        state$.quiz.selectedAnswer.set(answer);
+        state$.quiz.answered.set(true);
+        state$.quiz.waitingForOthers.set(true);
+        state$.quiz.displayQuestion.set(false);
+        // make request to BE to submit answer
```

Comment:
make sure to add this part
```
socket.emit(WS_EVENTS.ANSWER_QUESTION, roomId, playerId, answer);
```
from this PR https://github.com/yesh0907/Toohak-BE/pull/12